### DISCUSSION TITLE
Full packet capture

### DISCUSSION THREAD

**Discussion by gchumash** [Security-Onion-Solutions/securityonion#4370] [Unsupported Versions] [answered]
Hello,

I need to have FPC for sessions in my security onion environment to dig through during analysis. Is it possible to do FPC with SO? if yes, for how any days. 

Thanks
G

**Comment by dougburks** [comment, marked as answer]
Yes, Security Onion does full packet capture by default. The number of days of full packet capture will be dependent on how much disk space you have.

For more information, please see:
https://docs.securityonion.net/en/2.3/pcap.html
https://docs.securityonion.net/en/2.3/network.html
https://docs.securityonion.net/en/2.3/stenographer.html
